### PR TITLE
Added an image description for loops_image.png

### PR DESCRIPTION
**Pull request by spikelynch**
Added an image description for loops_image.png in 04-loops.md (issue #781)

### CODE REVIEW

File: _episodes/04-loop.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ for variable in collection:
 
 Using the oxygen example above, the loop might look like this:
 
-![loop_image](../fig/loops_image.png)
+![Graph illustrating a variable 'char' being assigned the value of each character in the word 'oxygen' in turn and then being printed](../fig/loops_image.png)
```
Comment (ldko) [inline _episodes/04-loop.md]: Based on [WebAIM's tip](https://webaim.org/techniques/alttext/) not to use "image of ..." or "graphic of ..." what do you say to replacing "Graph illustrating a" with "Loop" to read: "Loop variable 'char' being assigned the value of each character in the word 'oxygen' in turn and then being printed"?

Comment (spikelynch) [reply _episodes/04-loop.md:151]: Hi - yes, that's good.

File: _episodes/04-loop.md [hunk 2]
```diff
@@ -147,7 +147,7 @@ for variable in collection:
...
```
Comment (ldko) [inline _episodes/04-loop.md:150]:
```suggestion
![Loop variable 'char' being assigned the value of each character in the word 'oxygen' in turn and then being printed](../fig/loops_image.png)
```

Comment (maxim-belkin) [reply _episodes/04-loop.md]: What about changing `being` to `is` and clarifying that we're printing to the screen?

Comment (ldko) [reply _episodes/04-loop.md]: Also, see https://github.com/swcarpentry/python-novice-inflammation/pull/844 where @magsol has also proposed `alt` text for this image as well.

Comment (magsol) [reply _episodes/04-loop.md]: Sorry! I missed this one.

Comment (ldko) [reply _episodes/04-loop.md]: @spikelynch would you like to make the changes that Maxim suggests?

**Comment by ldko** [comment]
It seems I neglected to merge this one. Thanks for providing the alt text for this image @spikelynch -- it's much appreciated!

**Pull request merged**